### ISSUE TITLE
Investigate how to create a client flavour

### ISSUE THREAD

**Issue by MatMath** [MatMath/PagesBehindCouch#1]
##Client flavour
We want to have X client and each client have a different 
- Display structure
- Partials / Controller
- Some specific translation
- Page Logic.

How to make this work with Angular and Gulp.

**Comment by MatMath** [comment]
Some interesting info using Angular.material to read: https://gist.github.com/ryanpager/8696369ea540e250afcd

**Comment by MatMath** [comment]
Using *config module* with [recipe provider!](https://docs.angularjs.org/guide/providers#provider-recipe) could maybe work if we setup Gulp to handle the client switching.

**Comment by MatMath** [comment]
After many research and many possibility (
- Bower module per client, 
- Fancy config file at opening that build all,
- Each client in it's own app, 
We did a opt for a simple lazy way. 
Requirement is that is we have Client 1 code, we dont want to risk deploying to client 2 with the code of Client 1 and 2 inside the min.js file. 

We pushed the client folder outside the src code.
The client files and script are under a different module. 
We require that module to build the code. (That will lead to deployment pre-test to validate that Client 1 have client 1 module and not client 2 module)
Gulp task to copy the client module inside the
